### ISSUE TITLE
Plots

### ISSUE THREAD

**Issue by GiacobboNicola** [GiacobboNicola/checkedbox#4]
Generate a figure with:
* a plot with accuracy and loss as a function of epoch
* batch loss per epoch (line with alpha to show density) for both training set and validation set
